<?xml version="1.0"?>
<launch>
    <!-- roomba number -->
    <arg name="roomba" default="roomba1"/>

    <!-- launch node -->
    <!-- <node pkg="collaborative_vslam" type="scale_ratio_initializer_node" name="$(arg roomba)_init_ratio"> -->
    <node pkg="collaborative_vslam" type="scale_ratio_initializer_node" name="$(arg roomba)_init_ratio" output="screen">
        <!-- topic name -->
        <!-- sub -->
        <remap from="/visual_sign" to="/$(arg roomba)/orb_slam3/visualize_sign"/>
        <remap from="/lost_sign"   to="/$(arg roomba)/orb_slam3/lost_sign"/>
        <remap from="/pose"        to="/$(arg roomba)/orb_slam3/camera_pose"/>
        <remap from="/wheel_odom"  to="/$(arg roomba)/roomba/odometry"/>
        <!-- pub -->
        <remap from="/scale_ratio" to="/$(arg roomba)/scale_ratio"/>

        <!-- parameter -->
        <param name="/hz" value="15"/>
        <param name="/scale_ratio_th_percent" value="0.2"/>
        <param name="/duration_init" value="1.0"/>
    </node>
</launch>
